modules/example-apache-httpd-configuration.adoc

```diff
@@ -55,6 +57,35 @@ echo "Content-Type: application/json"
 echo ""
 echo '{"error": "Login failure"}'
 exit 0
+
+htpasswd -bc passwords basicauth-xxia1 redhat
+htpasswd -b passwords basicauth-xxia2 redhat
+
+oc create secret generic auth --from-file=passwords
+oc create cm cgi-bin --from-file=login.cgi --from-file=fail.cgi
+oc create cm login.conf --from-file=login.conf
+oc set volumes deployment/$APP --add -t secret --secret-name auth -m /etc/httpd/conf/auth
+oc set volumes deployment/$APP --add -t configmap --configmap-name cgi-bin -m /var/www/cgi-bin --default-mode 0777
+oc set volumes deployment/$APP --add -t secret --secret-name httpd-cert -m /etc/httpd/conf/tls/basic-auth
+oc set volumes deployment/$APP --add -t configmap --configmap-name login.conf -m /tmp/httpd
+----
+
+.Results from Example
+----
+oc get pod # Pod becomes Running now
+NAME                     READY   STATUS    RESTARTS   AGE
+httpd-7d4d97bc96-4kzgq   1/1     Running   0          9
+
+$ oc login -u basicauth-xxia1 -p redhat
+Login successful.
+
+$ oc config use-context admin
+$ oc get user
+NAME              UID                                    FULL NAME
+basicauth-xxia1   3884a251-660f-46a5-a081-e21d4923d79c   Hello basicauth-xxia1
+
+IDENTITIES
+basicauth-idp:basicauth-xxia1
```

Comment:
Same suggestion about self review on PR author side, did you find the output should be in single lines but you break to two lines?